pkg/platform/model/inventory.go

```diff
@@ -118,15 +118,19 @@ func SearchIngredientsStrict(language, name string) ([]*IngredientAndVersion, *f
 }
 
 func searchIngredients(limit int, language, name string) ([]*IngredientAndVersion, *failures.Failure) {
+	retry := retryhttp.New(retryhttp.DefaultClient)
+	defer retry.Close()
+
 	lim := int64(limit)
 
 	client := inventory.Get()
 
-	params := inventory_operations.NewGetNamespaceIngredientsParams()
+	params := inventory_operations.NewGetNamespaceIngredientsParamsWithContext(retry.Context)
+	params.SetHTTPClient(retry.Client.StandardClient())
+	params.SetTimeout(retry.Client.HTTPClient.Timeout)
```

Comment:
What's different about this `SetTimeout` ?

Reply:
I am about >85% sure that the context.Context is passed via "client operation" here: https://github.com/ActiveState/cli/blob/f715b239c228d260274d91f58b1ec0b60a6f704c/pkg/platform/api/inventory/inventory_client/inventory_operations/inventory_operations_client.go#L2263-L2275

And then is the first encountered (and used) context value here: https://github.com/go-openapi/runtime/blob/d5a301537e5208a4b0c720f36ba75c0fa802c7df/client/runtime.go#L411-L439

I thought it safest to set both to the same timeout in case something was ever wired differently, or my following of the code was incorrect.

Reply:
I traced it down to http://github.com/ActiveState/cli/blob/77e2701270db9b9789e00e1274af701a5fa1443c/vendor/github.com/go-openapi/runtime/client/runtime.go#L384-L384

Please use `params.SetTimeout(60)` and throw away the retryhttp stuff we did for this PR. Seems just for addressing this story only a single line change was requested.

I highly recommend you invest some time in setting up your vim with proper code intelligence or using an IDE that provides you with this. It's a HUGE time saver.